src/commands/sbt_cached.yml

```diff
@@ -50,6 +56,8 @@ steps:
               set +eo pipefail
               find $HOME/.sbt -name "*.lock" | xargs rm | true
               find $HOME/.ivy2 -name "ivydata-*.properties" | xargs rm | true
+              find -name target -exec tar -zcf ~/targets.tar.gz -H posix {} + | true
```

Comment:
A link to the sbt issue could be a good thing to add, so If they solve it we can know and remove the workaround.